omero/sysadmins/server-security.rst

```diff
@@ -21,7 +21,7 @@ client including:
 .. note:: The OMERO team treats the security of all components with care and
     attention. If you have a security issue to report, please do not hesitate
     to contact us using our private, secure mailing list as described on the
-    :secvuln:`security vulnerabilities <>` page.
+    :secvuln:`Security Advisories <>` page.
```

Comment:
This should actually link to the top-level  http://www.openmicroscopy.org/security/ page.

Reply:
In that case we will need a new link, we cannot use ``:secvuln:``